### ISSUE TITLE
Check for abstract base class definitions with no abstractmethods

### ISSUE THREAD

**Issue by Zac-HD** [PyCQA/flake8-bugbear#273]
```python
from abc import ABC, ABCMeta, abstractmethod

class Base(ABC):  # I'd like a lint error here if there are no @abstract* decorated methods

    # @abstractmethod
    def method(self):
        ...

class MyABC(metaclass=ABCMeta):  # and here too for the same reason
    pass
```

Suggested message: `{classname} is an abstract base class, but it has no abstract methods.  Remember to use @abstractmethod, @abstractclassmethod, and/or @abstractproperty decorators.`

We might also want to emit a warning for a probably-missing-decorator on any method of an ABC where the whole method body is `...`, even if there are some abstract-decorated methods on the class.

**Comment by cooperlees** [comment]
I can't see how this could draw false positives ... so can do nothing but help. I feel this fits things bugbear looks for ... PR will be accepted.

**Comment by jakkdl** [comment]
I'm starting on a PR now!

> We might also want to emit a warning for a probably-missing-decorator on any method of an ABC where the whole method body is ..., even if there are some abstract-decorated methods on the class.
How about extending this to the body being anything but `pass`, `...` or string constants - to also catch
```python
# @abstractmethod
def method(self):
  """my abstract method"""
  pass
```

**Comment by Zac-HD** [comment]
Closed by #274.

**Comment by jakkdl** [comment]
If you also want
> We might also want to emit a warning for a probably-missing-decorator on any method of an ABC where the whole method body is ..., even if there are some abstract-decorated methods on the class.

you can open a separate issue for it, should be an easy one as well :)

**Comment by Zac-HD** [comment]
Nah,  let's release what we've got now and let future features take care of themselves.

(though Cooper,  you might want to copy the auto release set up from flake8-async to make releases really easy in future?)

**Comment by cooperlees** [comment]
I'm not a fan of that tbh as we don't have good enough change log enforcement etc. with this project so the 1 commit + release via GitHub is there enough for me imo.
